src/fstests/runner.py

```diff
@@ -67,60 +77,52 @@ def _run_single_job(self, tarball_node, device):
         finally:
             return True
 
-    def check_kvm_xfstest_env(self):
+    def run(self, args):
+        sub_id = None
+        if not args.node_id:
+            sub_id = self._db.subscribe_node_channel({
+                'op': 'complete',
+                'name': 'tarball',
+            })
+            print('Listening for tarballs')
+            print('Press Ctrl-C to stop.')
         try:
-            result = subprocess.run(
-                "kvm-xfstests --help", check=True, shell=True)
-            if result.returncode == 0:
-                print(f"KVM-xftests found.")
-                return True
+            if sub_id:
+                while True:
+                    tarball_node = self._db.receive_node(sub_id)
+                    print(f"Node tarball with id: {tarball_node['_id']}\
+                    from revision: {tarball_node['revision']['commit'][:12]}")
```

Comment:
Please fix indentation, this looks like a new line of code.

Reply:
fixed